docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-immutable-con.adoc

```diff
@@ -0,0 +1,17 @@
+[id='environment-immutable-con']
+= Environment with immutable servers
+You can deploy an environment that includes several different pods running {KIE_SERVER} with preloaded processes. The database servers are, by default, also run in pods. Each {KIE_SERVER} pod can be separately scaled as necessary.
+
+In this case, any processes (KJAR files) must be loaded onto a {KIE_SERVER} at the time the image is created. You can not load or unload processes on a running {KIE_SERVER}. The advantage of this approach is that the {KIE_SERVER} with the processes in it runs like any other containerized service and does not require specialized management. You can use existing approaches to cloud integration to deploy and manage the servers.
+
+Optionally, you can also deploy a pod with {CENTRAL} Monitoring and a pod with Smart Router. You can use {CENTRAL} Monitoring to start and stop (but not deploy) processes on your {KIE_SERVERS} and to view monitoring data. 
+
+Smart Router is a single endpoint that can receive calls to any of your processes and route them automatically to the server that actually runs the process.
```

Comment:
"receive calls from any .." Right? Or do you mean intercept calls to any ... If you make a change, note that this description is also used elsewhere in your doc.

Reply:
No, it;s calls from client applications To any of the processes, and it's not really "intercept" as the router itself becomes the endpoint for the client. I'll put the client there.